# -*- coding: utf-8 -*-
# Generated by Django 1.11.8 on 2018-05-30 14:48
from __future__ import unicode_literals

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion
import django.utils.timezone


class Migration(migrations.Migration):

    initial = True

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
    ]

    operations = [
        migrations.CreateModel(
            name='DebugCategory',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('name', models.CharField(max_length=100)),
            ],
            options={
                'verbose_name': 'Exception Category',
                'verbose_name_plural': 'Exception Categories',
            },
        ),
        migrations.CreateModel(
            name='DebugRow',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('request_path', models.TextField(blank=True, null=True)),
                ('ex', models.TextField()),
                ('extra_ex', models.TextField(blank=True, null=True)),
                ('priority', models.IntegerField(choices=[(1, b'Very Low'), (2, b'Low'), (3, b'Medium'), (4, b'High'), (5, b'Very High')], default=3)),
                ('date', models.DateTimeField(default=django.utils.timezone.now)),
                ('read', models.BooleanField(default=False)),
                ('notes', models.TextField(blank=True, null=True)),
                ('fixed', models.BooleanField(default=False)),
                ('category', models.ForeignKey(default=1, on_delete=django.db.models.deletion.CASCADE, to='debugapp.DebugCategory')),
                ('user', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
            ],
            options={
                'verbose_name': 'Exception',
                'verbose_name_plural': 'Exceptions',
            },
        ),
    ]
